apps/client/src/register/components/registerForm.tsx

```diff
@@ -0,0 +1,143 @@
+import { Button } from '@mui/material';
+import FormInput from '../../components/FormInput';
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const RegisterForm: React.FC = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [passwordError, setPasswordError] = useState('');
+  const [confirmPasswordError, setConfirmPasswordError] = useState('');
```

Comment:
Too many useState here, I would suggest to use one object that contains all the props of the form, create a helper function that updates the field that you are changing and define an initial state for this, It's cleaner and also helps you repeat less code.